discovery-provider/src/utils/helpers.py

```diff
@@ -542,3 +541,8 @@ def get_final_poa_block() -> int:
         raise Exception("audius_final_poa_block not set")
 
     return final_poa_block
+
+
+def format_total_audio_balance(balance: str) -> str:
```

Comment:
I remember reading a doc about regular math being dangerous on Wei values because they are big integers. Do we have any utilities we are supposed to use in this codebase for formatting those values?

Reply:
good call

Reply:
spoke with @piazzatron , think we can just use python math here.